doc/coq.rst

```diff
@@ -304,21 +304,15 @@ coq.pp
 
 Authors of Coq plugins often need to write ``.mlg`` files to extend the Coq
 grammar. Such files are preprocessed with the ``coqpp`` binary. To help plugin
-authors avoid writing boilerplate, we provide a ``(coqpp ...)`` stanza:
+authors avoid writing boilerplate, we provide a ``(coq.pp ...)`` stanza:
 
 .. code:: scheme
 
-    (coq.pp (modules <mlg_list>))
+    (coq.pp
+     (modules <ordered_set_lang>))
```

Comment:
If I understand the code correctly, this should be a glob.

Reply:
It's easier to fetch the mlg sources using the predicate lang than it is with the ordered set one, so I would rather keep it like this.

Reply:
It is now ordered set lang behind the scenes.